octobot_trading/exchanges/config/exchange_config_data.py

```diff
@@ -114,27 +114,30 @@ def initialize_exchange_event_tree(self):
                     commons_enums.InitializationEventExchangeTopics.ORDER_BOOK,
                     commons_enums.InitializationEventExchangeTopics.FUNDING,
             ):
-                for symbol in self.traded_symbol_pairs:
-                    if topic in (
+                if not self.exchange_manager.is_backtesting:
+                    # disabled in backtesting: not all traded pairs are always available from the start:
+                    # don't wait for them in events
```

Comment:
I don't understand why we should not create these events. I think we should remove the "await event" in backtesting, not the event tree creation

Reply:
If we do that, it means that we will need to add a "if backtesting" everytime we want to use any of those events

Reply:
up!